paasta_tools/bounce_lib.py

```diff
@@ -339,19 +339,24 @@ def crossover_bounce(
     happy_new_tasks,
     old_app_live_happy_tasks,
     old_app_live_unhappy_tasks,
+    margin_factor=1.05,
```

Comment:
I know it increases the scope of the patch, but this really needs to be configurable via MarathonServiceConfig.